Mir module panics

This panic happens from time to time.

```
Mir node stopped with error: could not process PassiveModule (availability) events: module panicked: item with id '6' is not present in the ContextStore
Stack trace:
goroutine 4970354 [running]:
runtime/debug.Stack()
	/usr/local/go/src/runtime/debug/stack.go:24 +0x65
github.com/filecoin-project/mir.safelyApplyEventsPassive.func1()
	/home/circleci/go/pkg/mod/github.com/filecoin-project/mir@v0.0.0-20220829142836-04e7026a9bb8/workers.go:140 +0x6b
panic({0x3acfb20, 0xc0063a3e70})
	/usr/local/go/src/runtime/panic.go:838 +0x207
github.com/filecoin-project/mir/pkg/contextstore.(*sequentialContextStoreImpl[...]).Recover(0x203002?, 0x203002?)
	/home/circleci/go/pkg/mod/github.com/filecoin-project/mir@v0.0.0-20220829142836-04e7026a9bb8/pkg/contextstore/sequential.go:33 +0x95
github.com/filecoin-project/mir/pkg/dsl.Handle.RecoverAndRetainContext(...)
	/home/circleci/go/pkg/mod/github.com/filecoin-project/mir@v0.0.0-20220829142836-04e7026a9bb8/pkg/dsl/dslmodule.go:109
github.com/filecoin-project/mir/pkg/dsl.Handle.RecoverAndCleanupContext({0x30?}, 0xc008980ad8?)
	/home/circleci/go/pkg/mod/github.com/filecoin-project/mir@v0.0.0-20220829142836-04e7026a9bb8/pkg/dsl/dslmodule.go:117 +0x30
github.com/filecoin-project/mir/pkg/availability/batchdb/dsl.UponLookupBatchResponse[...].func1()
	/home/circleci/go/pkg/mod/github.com/filecoin-project/mir@v0.0.0-20220829142836-04e7026a9bb8/pkg/availability/batchdb/dsl/dsl.go:80 +0x77
github.com/filecoin-project/mir/pkg/availability/batchdb/dsl.UponEvent[...].func1()
	/home/circleci/go/pkg/mod/github.com/filecoin-project/mir@v0.0.0-20220829142836-04e7026a9bb8/pkg/availability/batchdb/dsl/dsl.go:61 +0x46
github.com/filecoin-project/mir/pkg/dsl.UponEvent[...].func1()
	/home/circleci/go/pkg/mod/github.com/filecoin-project/mir@v0.0.0-20220829142836-04e7026a9bb8/pkg/dsl/dslmodule.go:76 +0x4c
github.com/filecoin-project/mir/pkg/dsl.(*dslModuleImpl).ApplyEvents(0xc005103620, 0xc005a14ea0?)
	/home/circleci/go/pkg/mod/github.com/filecoin-project/mir@v0.0.0-20220829142836-04e7026a9bb8/pkg/dsl/dslmodule.go:148 +0x3a2
github.com/filecoin-project/mir/pkg/factorymodule.(*FactoryModule).forwardEvent(0xc0015e0b40, 0xc005a14ea0)
	/home/circleci/go/pkg/mod/github.com/filecoin-project/mir@v0.0.0-20220829142836-04e7026a9bb8/pkg/factorymodule/factory.go:171 +0xd9
github.com/filecoin-project/mir/pkg/factorymodule.(*FactoryModule).applyEvent(0x28?, 0xc008980d20?)
	/home/circleci/go/pkg/mod/github.com/filecoin-project/mir@v0.0.0-20220829142836-04e7026a9bb8/pkg/factorymodule/factory.go:79 +0x1e6
github.com/filecoin-project/mir/pkg/modules.ApplyEventsSequentially(0xc00854c490, 0xc008980d40)
	/home/circleci/go/pkg/mod/github.com/filecoin-project/mir@v0.0.0-20220829142836-04e7026a9bb8/pkg/modules/eventprocessing.go:24 +0xfe
github.com/filecoin-project/mir/pkg/factorymodule.(*FactoryModule).ApplyEvents(0xc00854c490?, 0xc005a14ea0?)
	/home/circleci/go/pkg/mod/github.com/filecoin-project/mir@v0.0.0-20220829142836-04e7026a9bb8/pkg/factorymodule/factory.go:57 +0x38
github.com/filecoin-project/mir.safelyApplyEventsPassive({0x459ba60?, 0xc0015e0b40?}, 0xc0015e0b40?)
	/home/circleci/go/pkg/mod/github.com/filecoin-project/mir@v0.0.0-20220829142836-04e7026a9bb8/workers.go:147 +0x79
github.com/filecoin-project/mir.(*Node).processModuleEvents(0xc00643c580, {0x45a9918, 0xc005d3c1c0}, {0x7fb5d8188688?, 0xc0015e0b40}, 0xc00767af00, 0xc00767ad20)
	/home/circleci/go/pkg/mod/github.com/filecoin-project/mir@v0.0.0-20220829142836-04e7026a9bb8/workers.go:93 +0x2db
github.com/filecoin-project/mir.(*Node).startModules.func1({0x3f4f2ee, 0xc}, {0x7fb5d8188688, 0xc0015e0b40}, 0xc006cd8000?)
	/home/circleci/go/pkg/mod/github.com/filecoin-project/mir@v0.0.0-20220829142836-04e7026a9bb8/node.go:331 +0x12b
created by github.com/filecoin-project/mir.(*Node).startModules
	/home/circleci/go/pkg/mod/github.com/filecoin-project/mir@v0.0.0-20220829142836-04e7026a9bb8/node.go:319 +0x24c

2022-08-30T06:56:27.776Z	INFO	mir-consensus	mir/manager.go:257	Mir manager shutting down	{"miner": "/root/t01003:t1kh77wai2ulyj25oxqciftm574ojboz3e6tuan7i"}
2022-08-30T06:56:27.776Z	INFO	mir-consensus	mir/manager.go:263	WAL closed
2022-08-30T06:56:27.776Z	INFO	mir-consensus	runtime/panic.go:838	Mir miner completed	{"addr": "t1kh77wai2ulyj25oxqciftm574ojboz3e6tuan7i"}
panic: miner consensus error: could not process PassiveModule (availability) events: module panicked: item with id '6' is not present in the ContextStore
Stack trace:
goroutine 4970354 [running]:
runtime/debug.Stack()
	/usr/local/go/src/runtime/debug/stack.go:24 +0x65
github.com/filecoin-project/mir.safelyApplyEventsPassive.func1()
	/home/circleci/go/pkg/mod/github.com/filecoin-project/mir@v0.0.0-20220829142836-04e7026a9bb8/workers.go:140 +0x6b
panic({0x3acfb20, 0xc0063a3e70})
	/usr/local/go/src/runtime/panic.go:838 +0x207
github.com/filecoin-project/mir/pkg/contextstore.(*sequentialContextStoreImpl[...]).Recover(0x203002?, 0x203002?)
	/home/circleci/go/pkg/mod/github.com/filecoin-project/mir@v0.0.0-20220829142836-04e7026a9bb8/pkg/contextstore/sequential.go:33 +0x95
github.com/filecoin-project/mir/pkg/dsl.Handle.RecoverAndRetainContext(...)
	/home/circleci/go/pkg/mod/github.com/filecoin-project/mir@v0.0.0-20220829142836-04e7026a9bb8/pkg/dsl/dslmodule.go:109
github.com/filecoin-project/mir/pkg/dsl.Handle.RecoverAndCleanupContext({0x30?}, 0xc008980ad8?)
	/home/circleci/go/pkg/mod/github.com/filecoin-project/mir@v0.0.0-20220829142836-04e7026a9bb8/pkg/dsl/dslmodule.go:117 +0x30
github.com/filecoin-project/mir/pkg/availability/batchdb/dsl.UponLookupBatchResponse[...].func1()
	/home/circleci/go/pkg/mod/github.com/filecoin-project/mir@v0.0.0-20220829142836-04e7026a9bb8/pkg/availability/batchdb/dsl/dsl.go:80 +0x77
github.com/filecoin-project/mir/pkg/availability/batchdb/dsl.UponEvent[...].func1()
	/home/circleci/go/pkg/mod/github.com/filecoin-project/mir@v0.0.0-20220829142836-04e7026a9bb8/pkg/availability/batchdb/dsl/dsl.go:61 +0x46
github.com/filecoin-project/mir/pkg/dsl.UponEvent[...].func1()
	/home/circleci/go/pkg/mod/github.com/filecoin-project/mir@v0.0.0-20220829142836-04e7026a9bb8/pkg/dsl/dslmodule.go:76 +0x4c
github.com/filecoin-project/mir/pkg/dsl.(*dslModuleImpl).ApplyEvents(0xc005103620, 0xc005a14ea0?)
	/home/circleci/go/pkg/mod/github.com/filecoin-project/mir@v0.0.0-20220829142836-04e7026a9bb8/pkg/dsl/dslmodule.go:148 +0x3a2
github.com/filecoin-project/mir/pkg/factorymodule.(*FactoryModule).forwardEvent(0xc0015e0b40, 0xc005a14ea0)
	/home/circleci/go/pkg/mod/github.com/filecoin-project/mir@v0.0.0-20220829142836-04e7026a9bb8/pkg/factorymodule/factory.go:171 +0xd9
github.com/filecoin-project/mir/pkg/factorymodule.(*FactoryModule).applyEvent(0x28?, 0xc008980d20?)
	/home/circleci/go/pkg/mod/github.com/filecoin-project/mir@v0.0.0-20220829142836-04e7026a9bb8/pkg/factorymodule/factory.go:79 +0x1e6
github.com/filecoin-project/mir/pkg/modules.ApplyEventsSequentially(0xc00854c490, 0xc008980d40)
	/home/circleci/go/pkg/mod/github.com/filecoin-project/mir@v0.0.0-20220829142836-04e7026a9bb8/pkg/modules/eventprocessing.go:24 +0xfe
github.com/filecoin-project/mir/pkg/factorymodule.(*FactoryModule).ApplyEvents(0xc00854c490?, 0xc005a14ea0?)
	/home/circleci/go/pkg/mod/github.com/filecoin-project/mir@v0.0.0-20220829142836-04e7026a9bb8/pkg/factorymodule/factory.go:57 +0x38
github.com/filecoin-project/mir.safelyApplyEventsPassive({0x459ba60?, 0xc0015e0b40?}, 0xc0015e0b40?)
	/home/circleci/go/pkg/mod/github.com/filecoin-project/mir@v0.0.0-20220829142836-04e7026a9bb8/workers.go:147 +0x79
github.com/filecoin-project/mir.(*Node).processModuleEvents(0xc00643c580, {0x45a9918, 0xc005d3c1c0}, {0x7fb5d8188688?, 0xc0015e0b40}, 0xc00767af00, 0xc00767ad20)
	/home/circleci/go/pkg/mod/github.com/filecoin-project/mir@v0.0.0-20220829142836-04e7026a9bb8/workers.go:93 +0x2db
github.com/filecoin-project/mir.(*Node).startModules.func1({0x3f4f2ee, 0xc}, {0x7fb5d8188688, 0xc0015e0b40}, 0xc006cd8000?)
	/home/circleci/go/pkg/mod/github.com/filecoin-project/mir@v0.0.0-20220829142836-04e7026a9bb8/node.go:331 +0x12b
created by github.com/filecoin-project/mir.(*Node).startModules
	/home/circleci/go/pkg/mod/github.com/filecoin-project/mir@v0.0.0-20220829142836-04e7026a9bb8/node.go:319 +0x24c


goroutine 4970067 [running]:
github.com/filecoin-project/lotus/chain/consensus/mir.Mine({0x45a9918, 0xc005d3c1c0}, {{0xc007650858?, 0x15?}}, {0x45d68c8, 0xc006be6000?})
	/home/circleci/project/chain/consensus/mir/mine.go:84 +0x1f93
github.com/filecoin-project/lotus/chain/consensus/hierarchical/subnet/consensus.Mine({0x45a9918?, 0xc005d3c1c0?}, {0x45d68c8?, 0xc006be6000?}, {{0xc007650858?, 0xc0045ede00?}}, 0xc001439260?)
	/home/circleci/project/chain/consensus/hierarchical/subnet/consensus/consensus.go:74 +0x15a
github.com/filecoin-project/lotus/chain/consensus/hierarchical/subnet/submgr.(*Subnet).mine.func2()
	/home/circleci/project/chain/consensus/hierarchical/subnet/submgr/subnet.go:294 +0x52
created by github.com/filecoin-project/lotus/chain/consensus/hierarchical/subnet/submgr.(*Subnet).mine
	/home/circleci/project/chain/consensus/hierarchical/subnet/submgr/subnet.go:293 +0x28f
```

--- a/pkg/availability/batchdb/fakebatchdb/fakebatchdb.go
+++ b/pkg/availability/batchdb/fakebatchdb/fakebatchdb.go
@@ -61,6 +61,7 @@ func NewModule(mc *ModuleConfig) modules.Module {
 		info, found := state.BatchStore[batchID]
 		if !found {
 			batchdbdsl.LookupBatchResponse(m, t.ModuleID(origin.Module), false, nil, nil, origin)
+			return nil
 		}
 
 		txs := make([]*requestpb.Request, len(info.txIDs))

